astrobee/config/communications/dds_ros_bridge.config

```diff
@@ -198,4 +198,6 @@ cameras = {
   {name="dock_cam", valid_resolutions={"1280_960", "1024_768", "640_480", "320_240"}, mode="FRAMES", max_frame_rate=15},
   {name="sci_cam", valid_resolutions={"1920_1080", "1280_720", "960_540", "480_270"}, mode="VIDEO", max_frame_rate=30},
   {name="haz_cam", valid_resolutions={"224_171"}, mode="FRAMES", max_frame_rate=5},
-  {name="perch_cam", valid_resolutions={"224_171"}, mode="FRAMES", max_frame_rate=5}}
+  {name="perch_cam", valid_resolutions={"224_171"}, mode="FRAMES", max_frame_rate=5},
+  {name="nav_cam_bayer", valid_resolutions={"224_171"}, mode="FRAMES", max_frame_rate=5},
```

Comment:
Is this the right thing to do? @kbrowne15 

Not sure we need to send the color images to the ground with dds? And if we do, it would not be the resolution of the perch cam?

Reply:
Yeah I'm totally down to take out the image_sampler part, and if we ever need it, add it back one day

Reply:
Yeah, let's leave this out for now. This is only a small part of what needs to be added to the bridge and I rather add everything at the same time.